hmda/src/test/scala/hmda/validation/rules/lar/validity/common/V627Spec.scala

```diff
@@ -4,9 +4,10 @@ import hmda.model.filing.lar.LarGenerators._
 import hmda.model.filing.lar.LoanApplicationRegister
 import hmda.validation.rules.EditCheck
 import hmda.validation.rules.lar.LarEditCheckSpec
+import hmda.census.records.CensusRecords
 
 class V627Spec extends LarEditCheckSpec {
-  override def check: EditCheck[LoanApplicationRegister] = V627
+  override def check: EditCheck[LoanApplicationRegister] = V627.withIndexedCounties(CensusRecords.indexedCounty2018)
```

Comment:
@kgudel does this mean  for 2019 we will reference 2018 census data across common validation checks and 2019 census data for edits unique to 2019?

Reply:
That 2018 data is just being used for the property tests for that edit. The running platform will use different data depending on what edit engine is called:
That 2018 engine code can be found [here](https://github.com/cfpb/hmda-platform/blob/5522e87f566e583dfc8ec665e51f71429962dd76/hmda/src/main/scala/hmda/validation/engine/2018/LarEngine2018.scala#L51-L53) and for the 2019 engine it is [here](https://github.com/cfpb/hmda-platform/blob/9293345f16920770d75e36e69587f4361d6ba135/hmda/src/main/scala/hmda/validation/engine/2019/LarEngine2019.scala#L52-L54).